services/webnn/ort/graph_builder_ort.cc

```diff
@@ -1781,9 +1778,9 @@ GraphBuilderOrt::AddGruOperation(const GruType& gru) {
                                           recurrent_weight_shape[1]}));
   }
   if (gru.layout == mojom::GruWeightLayout::kRzn) {
-    ASSIGN_OR_RETURN(weight, ConvertRnnGateLayout(weight, 3));
-    ASSIGN_OR_RETURN(recurrent_weight,
-                     ConvertRnnGateLayout(recurrent_weight, 3));
+    ASSIGN_OR_RETURN(weight, TransposeRnnWeightOrBiasLayout(weight, {1, 0, 2}));
```

Comment:
Define `{1, 0, 2}` as a constant array and use it for all occurrences, e.g.
```suggestion
    ASSIGN_OR_RETURN(weight, TransposeRnnWeightOrBiasLayout(weight, kZrnToRznPermutation));
```

Reply:
Done and named it `kRznToZrnPermutation`.